integration/container/copy_test.go

```diff
@@ -65,6 +65,34 @@ func TestCopyToContainerPathDoesNotExist(t *testing.T) {
 func TestCopyEmptyFile(t *testing.T) {
 	defer setupTest(t)()
 
+	ctx := context.Background()
+	apiclient := testEnv.APIClient()
+	cid := container.Create(ctx, t, apiclient)
+
+	// empty content
+	dstDir, _ := makeEmptyArchive(t)
+	err := apiclient.CopyToContainer(ctx, cid, dstDir, bytes.NewReader([]byte("")), types.CopyToContainerOptions{})
+	assert.NilError(t, err)
```

Comment:
Not a blocker; using `assert.Check()` allows the test to continue (`assert.NilError()` does a `t.Fatal()`, which in this case means we won't test the other cases if an earlier one fails)

```suggestion
	assert.Check(t, err)
```